services/web/server/tests/integration/projects/test_projects.py

```diff
@@ -235,3 +235,14 @@ def fake_template_projects_osparc(package_dir: Path) -> Dict:
         assert not error, pprint(error)
         # fake-template-projects.json + fake-template-projects.osparc.json
         assert len(projects) == (len(fake_template_projects) + len(fake_template_projects_osparc))
+
+async def test_project_uuid_uniqueness(loop, client, fake_project):
+    async with LoggedUser(client):
+        # create the project once
+        await _create_project(client, fake_project)
+        # create a second project with same uuid shall fail
+        with pytest.raises(AssertionError):

```

Comment:
Does alchemy really raise an AssertionError if it fails constraints? Is there some exception more concrete?